bin/server.rs

```diff
@@ -201,24 +200,55 @@ async fn main() -> anyhow::Result<()> {
         )
     })?;
 
-    tracing::log::info!(
-        "Starting server at {}, and serving bindles from {}",
-        addr.to_string(),
-        bindle_directory.display()
-    );
-
-    server(
-        store,
-        index,
-        bindle::authn::always::AlwaysAuthenticate,
-        bindle::authz::always::AlwaysAuthorize,
-        addr,
-        tls,
-        secret_store,
-        strategy,
-        keyring,
-    )
-    .await
+    #[cfg(not(feature = "embedded"))]
```

Comment:
I don't understand why we would decide this at compile time instead of at runtime. Wouldn't this require compiling multiple binary distributions of Bindle?

Reply:
Because it is experimental. I didn’t want to expose by default